source/content/guides/support/02-contact-support.md

```diff
@@ -48,15 +46,15 @@ The ticket support feature is available to certain Account packages and account
 
 After a ticket is submitted, you can view details for your support requests. If are you are part of an Workspace, your support tickets are visible to all members except [Unprivileged users](/guides/account-mgmt/workspace-sites-teams/teams#organizations-roles-and-permissions).
 
-## Real Time Chat Support
+## Live Chat
 
 Start a chat with our Support Team to ask questions or request assistance on issues within our [scope of support](#scope-of-support). This support feature is available to all users and sites across all plans, including Sandbox.
 
 To receive the highest level of support available to your site or Workspace, remember to access the chat from the Workspace if available.
 
 1. [Go to the workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces) or [Site Dashboard](/guides/account-mgmt/workspace-sites-teams/sites#site-dashboard) affected by the issue.
 
-1. Click **Support**, **Launch Chat**, then **New Conversation**.
+1. Go to the **Support** tab, then select **Start chat**.
```

Comment:
Keep the old verbiage here (Hermes), but we can change `New Conversation` to `Send us a message` (updated Intercom language)
![Screenshot 2024-08-22 at 2 51 12 PM](https://github.com/user-attachments/assets/c738f66a-b06b-448c-a514-6a0a9fe7e785)